### PR TITLE
Change Log for 1.2

### PR DESCRIPTION
**Pull request by unterstein**

### CODE REVIEW

**Comment by aquamatthias** [comment]
@sascala Can you have a look? This will be the release notes for 1.2

**Comment by sascala** [comment]
I made some light edits, @aquamatthias

**Comment by aquamatthias** [comment]
@sascala Thank you!

File: changelog.md [hunk 1]
```diff
@@ -1,8 +1,97 @@
...
+To upgrade to this version, stop all older Marathon instances before the new version starts.
+Otherwise, there is a risk of more than one leading master.
+
+
+#### Framework authentication command line flags
+Prior versions of Marathon have tried to authenticate whenever a principal has been provided via the command line.
+Framework authentication is now explicit. There is a command line toggle option for authentication: `--mesos_authentication`.
+This toggle is disabled by default. You must now supply this flag to use framework authentication.
+
+
```
Comment (gkleiman) [inline changelog.md]: We fixed the `unique service ports` validation. Even though we fixed a bug, it could be seen as a breaking change.

**Comment by gkleiman** [comment]
Two comments, even though this is already merged:

- We also added support for Docker `USER` networking.
- The "service ports must be unique" validation fix could be seen as a breaking change.

**Pull request merged**